extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -237,7 +237,8 @@ export class AmpA4A extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(unusedOnLayout) {
-    const preconnect = adPreconnect[this.element.getAttribute('type')];
+    const config = adConfig[this.element.getAttribute('type')];
+    const preconnect = config ? config.preconnect : null;
```

Comment:
@tdrl I plan to make the `_config.js` file have the whole list of supported networks, and throw error for unknown type. This eliminates nullable var and makes code easier and robust.